src/content/docs/learn/splashscreen.mdx

```diff
@@ -1,11 +1,253 @@
+
+<CTA/>
+
+- Project name: `splashscreen-lab`
+- Choose which language to use for your frontend: `Typescript / Javascript`
+- Choose your package manager: `pnpm`
+- Choose your UI template: `Vanilla`
+- Choose your UI flavor: `Typescript`
+- Would you like to setup the project for mobile as well? `yes`
+:::
+
+## Steps
+
+<Steps>
+
+1. ##### Install dependencies and run the project
+
+   Before you start developing any project it's important to build and run the initial template, just to validate your setup is working as intended.
+
+    <ShowSolution>
+    ```sh frame=none
+    # Make sure you're in the right directory
+    cd splashscreen-lab
+    # Install dependencies
+    pnpm install
+    # Build and run the app
+    pnpm tauri dev
+    ```
+    <Image src={step_1} alt="Successful run of the created template app."/>
+    </ShowSolution>
+
+1. ##### Register new windows in `tauri.conf.json`
+
+   The easiest way of adding new windows is by adding them directly to `tauri.conf.json`. You can also create them dynamically at startup,
+   but for the sake of simplicity lets just register them instead. Make sure you have a window with the label `main` that's being created as a hidden window and a window with the label `splashscreen` that's created as being shown directly. You can leave all other options as their defaults, or tweak them based on preference.
+
+    <ShowSolution>
+    ```json
+    // src-tauri/tauri.conf.json
+    {
+        "windows": [
+            {
+                "label": "main",
+                "visible": false
+            },
+            {
+                "label": "splashscreen",
+                "url": "/splashscreen"
+            }
+        ]
+    }
+    ```
+    </ShowSolution>
+
+1. ##### Create a new page to host your splashscreen
+
+   Before you begin you'll need to have some content to show. How you develop new pages depend on your chosen framework,
+   most have the concept of a "router" that handles page navigation which should work just like normal in Tauri, in which case
+   you just create a new splashscreen page. Or as we're going to be doing here, create a new `splashscreen.html` file to host the contents.
+
+   What's important here is that you can navigate to a `/splashscreen` URL and be shown the contents you want for your splashscreen. Try running the app again after this step!
+
+    <ShowSolution>
+    ```html
+    // /splashscreen.html
+    <!doctype html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" href="/src/styles.css" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Tauri App</title>
+    </head>
+    <body>
+        <div class="container">
+            <h1>Tauri used Splash!</h1>
+            <div class="row">
+                <h5>It was super effective!</h5>
+            </div>
+    </body>
```

Comment:
I think that you missed adding a closing tag for the parent `div` here.
```suggestion
            </div>
        </div>
    </body>
```

Reply:
That I did indeed 😅